.github/workflows/wheels.yml

```diff
@@ -77,19 +77,25 @@ jobs:
         - [windows-2019, win32]
         python: ["cp38", "cp39", "cp310"]
         include:
+        # TODO: uncomment PyPy 3.9 builds once PyPy
+        # re-releases a new minor version
+        # NOTE: This needs a bump of cibuildwheel version, also, once that happens.
```

Comment:
since cibuildwheel bumped the version, can you remove the "includes" here for PyPy and add "pp38", "pp39" to the pythons in line 78, and add ["excludes"](https://docs.github.com/en/actions/using-jobs/using-a-build-matrix-for-your-jobs#example-excluding-configurations-from-a-matrix) for the arm64 macos build

Reply:
arm64 wheels is not available as a platform for PyPy.
https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip

Reply:
Right. The "excludes" json section should remove a matrix combination, which would remove the need for the "includes"

Reply:
macos can build two variants: arm64 and x86_64. PyPy does not support the arm64 variant yet. PyPy _does_ support the aarch64 linux architecture.

Reply:
Ahh, cool. So you are not confused, I am. Sorry for the noise.